<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="1" name="docker_plugin_install_with_version" skipped="0" tests="1" time="4.46" timestamp="2022-03-15T18:12:29">
	<testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED"/>
		<system-out>18:12:25 INFO  ==============================================================
18:12:25 INFO  Id:    docker_plugin_install_with_version
18:12:25 INFO  Dir:   plugin_docker/
18:12:25 INFO  Title: Validate docker plugin install
18:12:25 DEBUG Execution order hint: 0.0
18:12:25 INFO  ==============================================================
18:12:25 DEBUG Process parameters for sudo&lt;docker_images&gt;
  command line : /usr/bin/sudo docker images
    arg #1     : docker
    arg #2     : images
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_images.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_images.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
18:12:25 DEBUG Waiting up to 600 secs for process sudo&lt;docker_images&gt; (pid 29309)
18:12:25 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
18:12:25 DEBUG Performing regex contains=False grep on file: /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_images.out
18:12:25 INFO  Grep on input file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_images.out ... passed
18:12:25 DEBUG --- test execute
18:12:25 DEBUG Process parameters for sudo&lt;plugin_install&gt;
  command line : /usr/bin/sudo /etc/tedge/sm-plugins/docker install registry --module-version 2.7.1
    arg #1     : /etc/tedge/sm-plugins/docker
    arg #2     : install
    arg #3     : registry
    arg #4     : --module-version
    arg #5     : 2.7.1
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/plugin_install.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/plugin_install.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
18:12:25 DEBUG Waiting up to 600 secs for process sudo&lt;plugin_install&gt; (pid 29319)
18:12:27 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
18:12:27 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:126,environment_docker_plugin.py:134,run.py:25]
18:12:27 DEBUG Process parameters for sudo&lt;docker_ps_latest&gt;
  command line : /usr/bin/sudo docker ps -q --latest
    arg #1     : docker
    arg #2     : ps
    arg #3     : -q
    arg #4     : --latest
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_ps_latest.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_ps_latest.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
18:12:27 DEBUG Waiting up to 600 secs for process sudo&lt;docker_ps_latest&gt; (pid 29348)
18:12:27 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
18:12:27 DEBUG --- test validate
18:12:27 DEBUG Process parameters for sudo&lt;docker_ps&gt;
  command line : /usr/bin/sudo docker ps
    arg #1     : docker
    arg #2     : ps
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_ps.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_ps.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
18:12:28 DEBUG Waiting up to 600 secs for process sudo&lt;docker_ps&gt; (pid 29358)
18:12:28 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
18:12:28 DEBUG Performing regex contains=True grep on file: /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_ps.out
18:12:28 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_ps.out contains &quot;registry:2.7.1&quot; ... failed [environment_docker_plugin.py:43,run.py:29]
18:12:28 DEBUG --- test cleanup
18:12:28 INFO  
18:12:28 INFO  cleanup:
18:12:28 DEBUG Running registered cleanup function: &lt;bound method DockerPlugin.cleanup_containers of &lt;DockerPluginInstallWithVersionTest object at 0x7607e6d0&gt;&gt;
18:12:28 INFO  Removing containers scheduled to be cleaned up: ['9f7da16d968b', '']
18:12:28 DEBUG Process parameters for sudo&lt;containers_for_cleanup&gt;
  command line : /usr/bin/sudo docker rm -f 9f7da16d968b ''
    arg #1     : docker
    arg #2     : rm
    arg #3     : -f
    arg #4     : 9f7da16d968b
    arg #5     : 
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/containers_for_cleanup.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/containers_for_cleanup.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
18:12:28 DEBUG Waiting up to 600 secs for process sudo&lt;containers_for_cleanup&gt; (pid 29367)
18:12:28 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
18:12:28 DEBUG Running registered cleanup function: &lt;bound method DockerPlugin.cleanup_images of &lt;DockerPluginInstallWithVersionTest object at 0x7607e6d0&gt;&gt;
18:12:28 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry', 'registry:2.7.1']
18:12:28 DEBUG Process parameters for sudo&lt;containers_for_cleanup&gt;
  command line : /usr/bin/sudo docker rmi -f registry registry registry:2.7.1
    arg #1     : docker
    arg #2     : rmi
    arg #3     : -f
    arg #4     : registry
    arg #5     : registry
    arg #6     : registry:2.7.1
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/containers_for_cleanup.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/containers_for_cleanup.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
18:12:28 DEBUG Waiting up to 600 secs for process sudo&lt;containers_for_cleanup&gt; (pid 29378)
18:12:29 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
18:12:29 DEBUG ProcessUser cleanup function done.
18:12:29 INFO  
18:12:29 INFO  Test duration: 4.46 secs
18:12:29 INFO  Test final outcome:  BLOCKED
18:12:29 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
18:12:29 INFO  
</system-out>
	</testcase>
</testsuite>
